bundles/org.openhab.binding.volvooncall/src/main/resources/ESH-INF/thing/vehicle.xml

```diff
@@ -0,0 +1,304 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="volvooncall"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Volvo Vehicle -->
+	<thing-type id="vehicle">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="vocapi" />
+		</supported-bridge-type-refs>
+
+		<label>Volvo Vehicle</label>
+		<!-- description>Thing for a Volvo Car</description-->
+
+		<channel-groups>
+			<channel-group typeId="doors" id="doors" />
+			<channel-group typeId="windows" id="windows" />
+			<channel-group typeId="odometer" id="odometer" />
+			<channel-group typeId="tank" id="tank" />
+			<channel-group typeId="position" id="position" />
+			<channel-group typeId="other" id="other" />
+			<channel-group typeId="lasttrip" id="lasttrip" />
+		</channel-groups>
+
+		<representation-property>vin</representation-property>
+
+		<config-description>
+			<parameter name="vin" type="text" required="true">
+				<label>Vehicle Identification Number</label>
+				<description>VIN of the vehicle associated with this Thing</description>
+			</parameter>
+
+			<parameter name="refresh" type="integer" min="5" required="false">
+				<label>Refresh interval</label>
```

Comment:
Can you make sure that every word from this label starts with a capital this is (more or less) the default in ESH / openHAB.